File: fuel management/prediction model/predict.py
from sklearn import preprocessing
import pickle
import sys

inputt=[[]]
for v in sys.argv[1:]:
  inputt[0].append(float(v))



x= inputt


model = pickle.load(open('D:/FYP/EW updated 2/controller/prediction model2/model.sav', 'rb'))
y=model.predict(x)

a=int(y)
b=str(a)
print(c)

    

File: backend/controllers/fuel management/predict.js
var {PythonShell} = require('python-shell');


exports.predict=async (req,res)=>{

  try{
    var options = {

      pythonOptions: ['-u'],
      args: req.body.inputt
    };

    PythonShell.run('D:/FYP/EW updated 2/controller/prediction model/predict.py', options, function (err, results) {
      if (err)
        throw err;

      console.log(results[0]);price:results[0]
      res.json({success:true})
    });
}
catch(err){
  res.json({ message: error.message,success:false });
}

}
